tensorflow/core/graph/mkl_layout_pass_test.cc

```diff
@@ -185,308 +225,432 @@ TEST_F(MklLayoutPassTest, Basic) {
 // Test set 1: Conv2D + AddBias
 
 // C=Conv2D(A,B); E=BiasAdd(C,D); Z=Zeta(E,Y)
-TEST_F(MklLayoutPassTest, NodeMerge_Conv2DWithBias_Positive) {
-  CHECK_EQ(kTensorOrdering, MklTfTensorOrdering::TENSORS_CONTIGUOUS);
-  InitGraph(
-      "node { name: 'A' op: 'Input'}"
-      "node { name: 'B' op: 'Input'}"
-      "node { name: 'C' op: 'Conv2D'"
-      " attr { key: 'T'                value { type: DT_FLOAT } }"
-      " attr { key: 'data_format'      value { s: 'NCHW' } }"
-      " attr { key: 'use_cudnn_on_gpu' value { b: false } }"
-      " attr { key: 'strides'          value { list: {i: 1, i:1, i:1, i:1} } }"
-      " attr { key: 'padding'          value { s: 'SAME' } }"
-      " attr { key: 'dilations'        value { list: {i: 1, i:1, i:1, i:1} } }"
-      " input: ['A', 'B']}"
-      "node { name: 'D' op: 'Input'}"
-      "node { name: 'E' op: 'BiasAdd'"
-      " attr { key: 'T'                value { type: DT_FLOAT } }"
-      " attr { key: 'data_format'      value { s: 'NCHW' } }"
-      " input: ['C', 'D'] }"
-      "node { name: 'Y' op: 'Input'}"
-      "node { name: 'Z' op: 'Zeta'"
-      " attr {key: 'T'                 value { type: DT_FLOAT } }"
-      " input: ['E', 'Y']}");
-  EXPECT_EQ(DoMklLayoutOptimizationPass(),
-            "A(Input);B(Input);D(Input);DMT/_0(Const);DMT/_1(Const);"
-            "DMT/_2(Const);E(_MklConv2DWithBias);Y(Input);Z(Zeta)|A->E;"
-            "A:control->DMT/_0:control;A:control->DMT/_1:control;"
-            "A:control->DMT/_2:control;B->E:1;D->E:2;DMT/_0->E:3;DMT/_1->E:4;"
-            "DMT/_2->E:5;E->Z;Y->Z:1");
-}
+#define REGISTER_TEST(NAME, T, INPUT)                                      \
+  TEST_F(MklLayoutPassTest, NAME##_##T) {                                  \
+    CHECK_EQ(kTensorOrdering, MklTfTensorOrdering::TENSORS_CONTIGUOUS);    \
+    InitGraph("node { name: 'A' op: '" #INPUT                              \
+              "'}"                                                         \
+              "node { name: 'B' op: '" #INPUT                              \
+              "'}"                                                         \
+              "node { name: 'C' op: 'Conv2D'"                              \
+              " attr { key: 'T'                value { type:" #T           \
```

Comment:
Nit: The orphan in the next line can actually be in this line. Would you mind making the changes? I think it's easier to read. Same for other places.
```suggestion
              " attr { key: 'T'                value { type:" #T " } }"    \
```

Reply:
@penpornk Thanks for review! This change was done automatically by clang_format tool. I, however, agree that removing the orphan braces looks much cleaner. But on the other hand, clang_format tool will do those changes every time we run it on this file. Let me know what you want me to do. I am ok either way.

Reply:
I see! Would you mind using `clang format off` around each `#define REGISTER_TEST`? It will skip `clang-format ` for that area.
```c++
// clang-format off
This part won't be formatted.
// clang-format on
```